tracim/setup.py

```diff
@@ -1,65 +1,87 @@
 # -*- coding: utf-8 -*-
-#quickstarted Options:
-#
-# sqlalchemy: True
-# auth:       sqlalchemy
-# mako:       True
-#
-#
+# quickstarted Options:
+#  sqlalchemy: True
+#  auth:       sqlalchemy
+#  mako:       True
 
-#This is just a work-around for a Python2.7 issue causing
-#interpreter crash at exit when trying to log an info message.
+import sys
+
+from setuptools.command.develop import develop
```

Comment:
by the way, maybe you can remove this part of code which is related to python 2.7 (not supported by tracim)